test/metabase/query_processor_test/explicit_joins_test.clj

```diff
@@ -853,7 +861,7 @@
 
 (deftest join-against-implicit-join-test
   (testing "Should be able to explicitly join against an implicit join (#20519)"
-    (mt/test-drivers (mt/normal-drivers-with-feature :left-join :expressions :basic-aggregations)
+    (mt/test-drivers (mt/normal-drivers-with-feature :left-join :expressions :basic-aggregations :foreign-keys)
       (mt/with-bigquery-fks #{:bigquery-cloud-sdk}
```

Comment:
Same thing as above, you're disabling the test for `:bigquery-cloud-sdk`

Reply:
I think the `:bigquery-cloud-sdk` driver declares `:foreign-keys` support.